contracts/tasks/helpers/constants.ts

```diff
@@ -46,11 +46,11 @@ export const NETWORKS_DEFAULT_GAS: Record<ESupportedChains, number | "auto"> = {
  * @returns {Record<ESupportedChains, string>} rpc urls for supported networks
  */
 export const getNetworkRpcUrls = (): Record<ESupportedChains, string> => {
-  const INFURA_KEY = process.env.INFURA_KEY ?? "";
+  const RPC_URL = process.env.RPC_URL ?? "";
 
   return {
-    [ESupportedChains.Sepolia]: `https://sepolia.infura.io/v3/${INFURA_KEY}`,
-    [ESupportedChains.OptimismSepolia]: `https://optimism-sepolia.infura.io/v3/${INFURA_KEY}`,
+    [ESupportedChains.Sepolia]: RPC_URL,
+    [ESupportedChains.OptimismSepolia]: RPC_URL,
```

Comment:
What do you think about adding network specific url?
For example: `SEPOLIA_RPC_URL` and `OPTIMISM_SEPOLIA_RPC_URL`. It will be more convenient to use with different networks and don't change `RPC_URL` everytime you need to deploy contracts or run tasks.

Reply:
Definitely, will fix that